Cargo.toml

```diff
@@ -13,21 +13,18 @@ crate-type = ["cdylib"]
 [profile.release]
 lto = true
 
-[dependencies.pyo3]
-version = "0.13.2"
-
 [features]
-extension-module = ["pyo3/extension-module"]
-default = ["extension-module"]
+default = ["sha2"]
```

Comment:
afaict from the docs, if you add `"extension-module"` to this list, it will be enabled by default.